lib/wallaby/query.ex

```diff
@@ -217,6 +278,18 @@ defmodule Wallaby.Query do
   end
 
   @spec compile(t) :: compiled
+  @doc """
+  Takes a query and compiles it into either css or xpath, which are the queries
+  that will finally be issued to the driver.
```

Comment:
How about "Compiles a query into css or xpath so its ready to be sent to the driver"?